lib/cog_api/resources/trigger.ex

```diff
@@ -1,13 +1,14 @@
 defmodule CogApi.Resources.Trigger do
   @derive [Poison.Encoder]
 
-  defstruct [:id,
-             :name,
-             :pipeline,
-             :as_user,
-             :timeout_sec,
-             :description,
-             :invocation_url]
+  defstruct [id: nil,
```

Comment:
You can move the keywords to the bottom of the list and leave the others as atoms. I don't have a strong opinion on it just seems weird to explicitly define things as `nil`.